jstz_cli/src/repl.rs

```diff
@@ -36,7 +36,13 @@ pub fn exec(self_address: Option<String>, cfg: &Config) -> Result<()> {
 
     let realm_clone = rt.realm().clone();
 
-    realm_clone.register_api(ConsoleApi, rt.context());
+    realm_clone.register_api(
+        ConsoleApi {
+            contract_address: address.clone(),
+            operation_hash: Default::default(),
+        },
+        rt.context(),
+    );
```

Comment:
I'd prefer to avoid the use of `Default::default()` here. I'm trying to think of a better approach to avoid dummy values in `jstz_api` for the CLI (since it is a code smell). 

We could try having `ConsoleApi` as an enum which is either `Cli` or `Proto { contract_address, operation_hash }`?